webapp-django/crashstats/settings/base.py

```diff
@@ -177,18 +176,33 @@ def path(*dirs):
 # Always generate a CSRF token for anonymous users.
 ANON_ALWAYS = True
 
-LOG_LEVEL = logging.INFO
+LOGGING_LEVEL = config('LOGGING_LEVEL', 'INFO')
 
-HAS_SYSLOG = True  # XXX needed??
-
-LOGGING_CONFIG = None
-
-# This disables all mail_admins on all django.request errors.
-# We can do this because we use Sentry now instead
 LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'level': LOGGING_LEVEL,
+            'class': 'logging.StreamHandler',
+        },
+    },
     'loggers': {
+        'django': {
+            'handlers': ['console'],
+            'level': LOGGING_LEVEL,
+        },
+        # This disables all mail_admins on all django.request errors because we
+        # use Sentry for that
         'django.request': {
-            'handlers': []
+            'handlers': ['console'],
+        },
```

Comment:
This was in there before, but assuming I'm right about `LOGGING_CONFIG`, it was ignored and had no effect.

I've never had to disable the `mail_admins` thing in a Django project before. I'm not sure how to verify whether we need this or not in a local development environment. Do you think we want this or can I nix it?

Reply:
Awesome! Good to know. I'll nix the comment.